source/about/mattermost-v9-changelog.md

```diff
@@ -32,11 +32,17 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
-     - Note: The value cannot be dynamically changed from the System Console while the server is running. One needs to shut down the server, manually edit the config, and then restart the server when switching from one backend to another, and either use mmctl or edit the config manually.
-     - If you are using Opensearch, you **must** set the backend to opensearch. Otherwise Mattermost will not work. 
-     - If you are using Elasticsearch v8, be sure to set ``action.destructive_requires_name`` to ``false`` in ``elasticsearch.yml`` to allow for wildcard operations to work.
-     - For AWS customers on Opensearch, do not enable "compatibility mode".
+ - Added support for Elasticsearch v8. Also added Beta support for [Opensearch](https://opensearch.org/). A new config setting ``ElasticsearchSettings.Backend`` has been added to differentiate between Elasticsearch and Opensearch. The default value is `elasticsearch`, which breaks support for AWS Elasticsearch v7.10.x since the official v8 client only works from Elasticsearch v7.11 and higher versions. See the important note below for details.
+ - Mattermost supports Elasticsearch v7.17+. However, we recommend upgrading your Elasticsearch v7 instance to v8.x. See the [Elasticsearch upgrade](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html) documentation for details.
+ - When using Elasticsearch v8, ensure you set ``action.destructive_requires_name`` to ``false`` in ``elasticsearch.yml`` to allow for wildcard operations to work.
+
+```{Important}
+If you're using AWS Elasticsearch, you must:
+- Upgrade to AWS Opensearch for future compatibility. Refer to the [Opensearch upgrade](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/version-migration.html) documentation for details.
+- Shut down the Mattermost server and update the Mattermost `ElasticsearchSettings.Backend` configuration setting value from `elasticsearch` to `opensearch` manually or using [mmctl](https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-config-set). This value cannot be  changed using the System Console nor changed dynamically while the server is running. See the Mattermost [Elasticsearch backend type](https://docs.mattermost.com/configure/environment-configuration-settings.html#elastic-backendtype) documentation for additional details.
+- Restart the Mattermost server after updating the configuration setting value.
+- Disable "compatibility mode" in Opensearch.
```

Comment:
@agnivade should this be done before the server restart?

Reply:
Technically it can be done any time. If you do it after restart, then you need to click on "test connection" again to re-initialize the client. So for the sake of simplicity, let's recommend to do it before.